package.json

```diff
@@ -9,7 +9,8 @@
   },
   "dependencies": {
     "loader-utils": "^1.0.2",
-    "mime": "1.3.x"
+    "mime": "1.3.6",
```

Comment:
why without `^`, this will force us to constantly update and spend for this time :disappointed: